consent-ui/Dockerfile

```diff
@@ -1,3 +1,23 @@
+#############
+### build ###
+#############
+
+# base image
+FROM node:12.14.0 as build-stage
```

Comment:
Why is this change needed? If you want to avoid local UI build or embed docker-compose build use different Dockerfile